osu.Game.Rulesets.Osu/Difficulty/OsuPerformanceCalculator.cs

```diff
@@ -76,13 +76,7 @@ protected override PerformanceAttributes CreatePerformanceAttributes(ScoreInfo s
                 }
                 else
                 {
-                    double fullComboThreshold = attributes.MaxCombo - countSliderEndsDropped;
-
-                    if (scoreMaxCombo < fullComboThreshold)
-                        effectiveMissCount = fullComboThreshold / Math.Max(1.0, scoreMaxCombo);
-
-                    // Combine regular misses with tick misses since tick misses break combo as well
-                    effectiveMissCount = Math.Min(effectiveMissCount, countSliderTickMiss + countMiss);
+                    effectiveMissCount = countMiss;
```

Comment:
What is your reason for changing this?

Reply:
the usage of estimators for lazer non-CL scores was added as a bandaid to nerf oshama scramble without merging this change